docs/release-notes/14-1-0/README.md

```diff
@@ -16,16 +16,20 @@ We released [OpenProject 14.1.0](https://community.openproject.org/versions/2030
 
 ### PDF export of Gantt view, e.g. for printing (Enterprise add-on)
 
-It is now possible to export a Gantt view as PDF in the OpenProject Enterprise edition. This allows users to print Gantt charts. Choose between different paper formats, set the colomn width, and specify whether the PDF file should display your work packages on a daily, monthly or quarterly basis.
+It is now possible to export a Gantt view as PDF in the OpenProject Enterprise edition. This allows users to easily print Gantt charts in a nice design. Choose between different paper formats, set the column width, and specify whether the PDF file should display your work packages on a daily, monthly or quarterly basis.
+
+Please note that we plan to further improve the PDF export of Gantt view in future releases.
 
 ![Export your Gantt view as PDF](openproject-14-1-gantt-pdf-export-figma.png)
 
 ### Favorite projects
 
-This feature was requested by many in the Community and will be a great benefit, especially for organizations with many projects: OpenProject 14.1 allows you to add projects to favorites so that they can be found more quickly. We recently gave an overview in our blog of how exactly you can add projects to favorites or remove them from favorites – and where you can display them.
+This feature was requested by many in the Community and will be a great benefit, especially for organizations with many projects: OpenProject 14.1 allows you to add projects to favorites so that they can be found more quickly. We [recently gave an overview in our blog](https://www.openproject.org/blog/favorite-projects/) of how exactly you can add projects to favorites or remove them from favorites – and where you can display them.
 
 ![OpenProject dropdown menu showing favorite projects, displayed with yellow stars](openproject-14-1-favorite-projects-overview.shadow.png)
 
+See our documentation to learn more about [how to mark a project as favorite](https://www.openproject.org/docs/user-guide/project-overview/#mark-a-project-as-favorite).
+
 ### Advanced features for the Meetings module
 
 The Meetings module is currently being continuously improved. With OpenProject 14.1...
```

Comment:
Did you want to mention previous changes to the Meetings Module here?

Reply:
Not in detail, I just wanted to show that it is a focus topic for us right now